zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -715,8 +766,13 @@ def flow(  # pylint:disable=too-many-branches, too-many-statements
             )
             yield tasks
             yield mazepa.Dependency()
+            reducer: ReduceOperation
+            if self.processing_blend_mode == "max":
+                reducer = ReduceByMaxValue()
+            else:
+                reducer = ReduceByWeightedSum()
```

Comment:
Change to one line:
`reducer = ReduceByMaxValue() if self.processing_blend_mode =="max" else ReduceByWeightedSum()`? I guess it might be a little long.

Reply:
Yeah I had it that way first, but one of our commit checks forced the line break, at which point I thought the version above was cleaner.